script.js

```diff
@@ -0,0 +1,145 @@
+function showBlock(){
+	fetch('./mock.json')
+    .then(response => response.json())
+    .then(data => {
+        addDataToPage(data);
+    });
+	
+}
+
+function showBlockPromise() {
+	makeRequest('GET', 'mock.json')
+		.then(function(e) {
+			addDataToPage(JSON.parse(e.target.response));
+		}, function(e) {
+			throw e;
+		});
+}
+
+function makeRequest(method, url) {
+	return new Promise(function (resolve, reject) {
+        var xhr = new XMLHttpRequest();
+        xhr.open(method, url);
+        xhr.onload = resolve;
+        xhr.onerror = reject;
+        xhr.send();
+    });
+}
+
+function clearBlock() {
+	document.getElementById('data-box').innerHTML = '';
+}
+
+function takeDataFromAPI() {
+	let selectBooks = document.getElementById('select1').value;
+	if(selectBooks === null || selectBooks === ""){
+		alert('Please, enter request');
```

Comment:
Don't use alerts

Reply:
if(selectBooks === null || selectBooks === "") return;
in this case you don't need else